l1-contracts/contracts/zksync/facets/Executor.sol

```diff
@@ -433,7 +433,14 @@ contract ExecutorFacet is Base, IExecutor {
     }
 
     function _batchMetaParameters() internal view returns (bytes memory) {
-        return abi.encodePacked(s.zkPorterIsAvailable, s.l2BootloaderBytecodeHash, s.l2DefaultAccountBytecodeHash);
+        return
+            abi.encodePacked(
+                s.zkPorterIsAvailable,
+                s.l2BootloaderBytecodeHash,
+                s.l2DefaultAccountBytecodeHash,
+                // VM 1.5.0 requires us to pass the EVM simulator code hash. For now it is 0.
+                bytes32(0)
```

Comment:
Brrr, so we should commit batches and verify all of them before upgrade? :(

Reply:
Sure, even without this change, `Verifier` would change and so that's why all the batches have to be verified